cypress/support/helper.js

```diff
@@ -1,6 +1,36 @@
-import { knobsTab } from "../locators/commonLocators";
+import { knobsTab } from "../locators/commonLocators"
+import { draggableRecordByText } from "../locators/draggableContextLocators"
+import { DEBUG_FLAG } from ".";
 
-export function visitComponentUrl(url) {
-    cy.visit(Cypress.env(url))
-    knobsTab().click()
-}
\ No newline at end of file
+export function visitComponentUrl(component, suffix = 'default', iFrameOnly = false) {
+    cy.visit(prepareUrl(component, suffix, iFrameOnly))
+    if(!iFrameOnly) knobsTab().click()
+}
+
+export function dragAndDropForDraggableRecord(record, position) {
```

Comment:
I'm wondering whether or not we could make this function more general, so it can be reused for other components / extended for dragging along the `X` axis.

Reply:
yes indeed, current sprint I am working on Configurable Items so I will make that method more general and reuse it

Reply:
I made general method for cypress/FE-1426 branch